src/v0/service_category.ts

```diff
@@ -62,8 +62,9 @@ export class ServiceCategory extends ThBaseHandler {
     let next
 
     try {
-      const base = this.uriHelper.generateBaseUri()
-      const uri = this.uriHelper.generateUriWithQuery(base, query)
+      const localUriHelper = new UriHelper('/api/v0/product_groups', this.options)
+      const base = localUriHelper.generateBaseUri()
+      const uri = localUriHelper.generateUriWithQuery(base, { is_service_category: true, ...query })

```

Comment:
no, that shouldn't be a part of the SDK
Just use existing `productGroups().getAll()` and pass `is_service_category: true` inside query param